app/models/reservation.rb

```diff
@@ -1,2 +1,5 @@
 class Reservation < ApplicationRecord
+  has_many :reservations
```

Comment:
# reviews

Reply:
You are right, should be :reviews instead